### PR TITLE
POL489 example of AWS STS Multi Account Cred

### PR DESCRIPTION
**Pull request by cdwilhelm**
### Description

pol489

### Issues Resolved

[List any existing issues this PR resolves]

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

**Review by XOmniverse — DISMISSED** [review]
LGTM

File: compliance/aws/disallowed_regions/README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ This policy checks all instances in a set of disallowed regions. The user is giv
 ## Input Parameters
 
 - *Email addresses to notify* - Email addresses of the recipients you wish to notify when new incidents are created.
+- *Account Number* - The Account number for use with the AWS STS Cross Account Role.  Leave blank when using AWS IAM Access key and secret. [more](https://docs.flexera.com/flexera/EN/Automation/ProviderCredentials.htm#automationadmin_1982464505_1123608)
```
Comment (Haydenmarx) [inline compliance/aws/disallowed_regions/README.md:15]: Double space between period and new sentence. Looks like we're using single space between sentences in other places.

I'm not sure how much space we want to devote, but I'd want to include a little bit of the logic about when the user would want to pass. Maybe something like this:

- *Account Number* - The Account number for use with the AWS STS Cross Account Role. Leave blank when using AWS IAM Access key and secret. It only needs to be passed when the desired AWS account is different than the one associated with the Flexera One credential. [more](https://docs.flexera.com/flexera/EN/Automation/ProviderCredentials.htm#automationadmin_1982464505_1123608)

File: compliance/aws/iam_role_audit/aws_iam_role_audit.pt [hunk 2]
```diff
@@ -21,6 +21,12 @@ parameter "param_email" do
   description "A list of email addresses to notify."
 end
```
Comment (Haydenmarx) [inline compliance/aws/iam_role_audit/aws_iam_role_audit.pt:23]: missing version bump

File: compliance/aws/instances_without_fnm_agent/aws_instances_not_running_flexnet_inventory_agent.pt [hunk 3]
```diff
@@ -71,6 +71,7 @@ credentials "auth_aws" do
   label "AWS"
   description "Select the AWS Credential from the list."
   tags "provider=aws"
+  aws_account_number $param_aws_account_number
```
Comment (Haydenmarx) [inline compliance/aws/instances_without_fnm_agent/aws_instances_not_running_flexnet_inventory_agent.pt:74]: The new parameter isn't set

File: cost/aws/rds_instance_license_info/rds_instance_license_info.pt [hunk 4]
```diff
@@ -28,6 +28,12 @@ parameter "param_allowed_regions" do
   description "A list of allowed regions. See the README for more details"
 end
 
+parameter "param_aws_account_number" do
+  type "string"
+  label "Account Number"
+  description "The account number for AWS STS Cross Account Roles."
+end
+
```
Comment (Haydenmarx) [inline cost/aws/rds_instance_license_info/rds_instance_license_info.pt:36]: needs version bump from 2.6 to 2.7

File: compliance/aws/scp_audit/aws_scp_audit.pt [hunk 5]
```diff
@@ -21,6 +21,12 @@ parameter "param_email" do
   description "A list of email addresses to notify."
 end
 
+parameter "param_aws_account_number" do
```
Comment (Haydenmarx) [inline compliance/aws/scp_audit/aws_scp_audit.pt:24]: Needs version bump

**Review by Haydenmarx — APPROVED** [review]
lgtm

**Pull request merged**